src/accountsdb/db.zig

```diff
@@ -40,6 +40,7 @@ const RefMemoryLinkedList = _accounts_index.RefMemoryLinkedList;
 const AccountRef = _accounts_index.AccountRef;
 const DiskMemoryAllocator = _accounts_index.DiskMemoryAllocator;
 
+pub const DB_PROGRESS_UPDATES_NS = 5 * std.time.ns_per_s;
```

Comment:
I think you once mentioned a concern about clogging up the logs with too many periodic messages. We could expose log intervals for this kind of thing with a centrally managed configuration, for example CLI options. For now this looks good, just something to think about.

Reply:
yeah agreed - would be nice to move these to something more easily configurable